calcom/cal.com
# Prevent saving error responses from Zoom to user credentials

## What does this PR do?



The purpose of this PR is to prevent error responses from Zoom to be saved under a user's credentials. This would wipe out the refresh token which prevents a user from ever getting a valid token again.

In the future, I want to do an audit of our all our apps to make sure this isn't happening.

Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change





## How should this be tested?



- Add Zoom to the account
- In the DB, find the zoom credential and delete a few characters from the refresh token and change the expiration date
- Try and create an event with Zoom as the location, you should receive an error but the credential's should still exist
